Library/Homebrew/test/.simplecov

```diff
@@ -14,14 +14,23 @@ SimpleCov.start do
   add_filter "Homebrew/vendor/"
 end
 
-if name = ENV["HOMEBREW_INTEGRATION_TEST"]
-  SimpleCov.command_name "brew #{name}"
+module Homebrew
+  class NullFormatter
+    def format(_result)
+      # Avoid generating intermediate `index.html` to save some time.
+    end
+  end
+end
+
+if ENV["HOMEBREW_INTEGRATION_TEST"]
+  SimpleCov.command_name ENV["HOMEBREW_INTEGRATION_TEST"]
   SimpleCov.at_exit do
     exit_code = $!.nil? ? 0 : $!.status
     $stdout.reopen("/dev/null")
     SimpleCov.result.format!
```

Comment:
You're right! Calling `.result` is already sufficient and there's no need for a dummy formatter. Short and simple solution. :smiley_cat: